root/tree/cloning/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
 if(ROOTTEST_DIR)
       set(ROOT_EVENT_DIR ${ROOTTEST_DIR}/root/treeformula/event/)
+      set(ROOT_DIR ${ROOTSYS})
+      execute_process(COMMAND root-config --tutdir OUTPUT_VARIABLE ROOT_TUTDIR OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
This command seems identical, whether `ROOTTEST_DIR` or not. Move outside the `if()`?